docs/src/pages/system/basics/Demo.js

```diff
@@ -21,11 +17,21 @@ export default function Demo() {
         fontWeight: 'fontWeightBold',
       }}
     >
-      <Img
-        sx={{ width: '100%', maxWidth: { xs: 350, md: 250 } }}
-        alt="The house from the offer."
-        src="https://images.unsplash.com/photo-1512917774080-9991f1c4c750?auto=format&w=400&dpr=2"
-      />
+      <Box
+        sx={{
+          width: '100%',
+          maxHeight: { xs: 233, md: 167 },
+          maxWidth: { xs: 350, md: 250 },
+        }}
+        clone
+      >
+        <img
+          alt="The house from the offer."
+          height="233"
+          width="350"
```

Comment:
It creates a distortion

<img width="299" alt="Capture d’écran 2020-11-13 à 20 50 14" src="https://user-images.githubusercontent.com/3165635/99114927-ea1c4780-25f1-11eb-9f1e-c2b847e274f9.png">

Reply:
Fixed. It's now getting slightly cropped but the window delta where this happens is maybe 20px wide so barely noticeable. Still not optimal but better than the current state.